src/main/java/org/kitteh/irc/client/library/implementation/EventListener.java

```diff
@@ -977,6 +977,12 @@ public void ctcp(ClientReceiveCommandEvent event) {
                         case "FINGER":
                             reply = "FINGER om nom nom tasty finger";
                             break;
+                        case "DCC":
```

Comment:
Sounds like bad coding practice to me. I thought the primary use of that was to make more than one case statement apply.